libraries/standard/http/cbmc/proofs/demo/Makefile

```diff
@@ -0,0 +1,82 @@
+METHOD=IotMqtt_Disconnect
+ENTRY=harness
+
+################################################################
+# Proof assumptions
+
+# Bound the number of subscriptions in a list (BOUND = MAX-1)
+SUBSCRIPTION_COUNT_MAX=2
+DEF += -DSUBSCRIPTION_COUNT_MAX=$(SUBSCRIPTION_COUNT_MAX)
+
+# allow mqtt mallocs to fail for coverage
+DEF += -include mqtt_state.h
+DEF += -DIotMqtt_MallocMessage=malloc_can_fail
+DEF += -DIotMqtt_MallocOperation=malloc_can_fail
+
+################################################################
+
+SRCDIR ?= $(abspath $(PROOFDIR)/../../..)
+
+INC += \
+	-I$(SRCDIR)/cbmc/proofs \
+	-I$(SRCDIR)/demos \
+	-I$(SRCDIR)/libraries/ \
+	-I$(SRCDIR)/libraries/platform \
+	-I$(SRCDIR)/libraries/standard/common/include \
+	-I$(SRCDIR)/libraries/standard/mqtt/include \
+	-I$(SRCDIR)/libraries/standard/mqtt/src \
+	-I$(SRCDIR)/libraries/standard/mqtt/src/private \
+	-I$(SRCDIR)/ports/common/include \
+
+DEF += \
+        -DIOT_SYSTEM_TYPES_FILE=\"$(SRCDIR)/ports/posix/include/iot_platform_types_posix.h\" \
+        -Diotmqtt_EXPORTS \
+
+COMPILE_FLAGS = -std=gnu99
```

Comment:
In other projects, we use `+=` for everything, even for things that are defined for the first time. This makes it easy to add additional compile flags when invoking make.